src/frontend/screens/Library/components/GameCard/index.css

```diff
@@ -10,6 +10,11 @@
   flex-direction: column;
   border-radius: var(--space-3xs);
   animation: fade-in 0.2s ease-in;
+  aspect-ratio: 173/275;
+}
+
+.gameCard.gamepad {
+  aspect-ratio: 3/4;
```

Comment:
these are the current ratios, but I'm hardcoding them so the empty divs use the correct space